app/src/main/res/layout/dashboard_page.xml

```diff
@@ -106,6 +106,34 @@
 
                 <!-- Your content for the second CardView (e.g., icon, title, etc.) -->
 
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:orientation="vertical"
+                    android:padding="8dp">
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="Today Sales"
+                        android:textSize="11sp"
+                        android:textStyle="bold"
+                        />
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="*For past 24 hours"
+                        android:textSize="7sp"
+                        />
```

Comment:
No need to indicate "For past 24hrs", this is already understood from the title itself "Today Sales"